### ISSUE TITLE
Relaxed Hanami dependency for new projects

### ISSUE THREAD

**Issue by jodosha** [hanami/hanami#501]
## Actual

We generate new projects with a strict dependency on `hanami` gem.

```ruby
# Gemfile
gem 'hanami', '0.8.0'
```

## Proposed

Because we follow SemVer, we can relax the dependency. In case of a patch release, developers can update their apps via a simple `bundle update hanami`, **instead of edit their `Gemfile`**.

New generated projects should have this `Gemfile`.

```ruby
# Gemfile
gem 'hanami', '~> 0.8'
```

---

/cc @hanami/contributors WDYT?

**Comment by pascalbetz** [comment]
+1

**Comment by accuser** [comment]
Better to make use of semantic versioning and relax the dependency. If anyone wants to freeze the gem dependency, they will know what they are doing and why.

:+1:

**Comment by joneslee85** [comment]
Me like. Me plus one

**Comment by hlegius** [comment]
:+1:

**Comment by jodosha** [comment]
Solved by #503
